### PR TITLE
feat: updated to miniTDF conventions. Adapted test names accordingly

### PR DESCRIPTION
**Pull request by sebastian-wehner-bruker**
For the release of ProteoScape 2024 the naming conventions for miniTDF were changed to more accurately represent what the files contain. This updated naming convention is included in this PR for miniTDF spectrum reading. The test-file-names were also adapted accordingly.

### CODE REVIEW

**Review by sander-willems-bruker — CHANGES_REQUESTED** [review]

**Comment by sander-willems-bruker** [comment]
We can consider doing some if these checks in file_format. For completeness, we should perhaps also enforce the existence of regular "analysis.tdf" and "_bin" files rather than just looking at the extension

File: src/file_readers/spectrum_readers/mini_tdf_reader.rs [hunk 1]
```diff
@@ -23,6 +24,18 @@ pub struct MiniTDFReader {
     frame_reader: BinFileReader,
 }
 
+fn find_ms2spectrum_file(ms2_dir_path: &str, extension: String) -> String {
+    let files = fs::read_dir(ms2_dir_path).unwrap();
+    for file in files {
+        let filename = file.unwrap().path().file_name().unwrap().to_str().unwrap().to_owned();
+        if filename.ends_with( std::format!("ms2spectrum.{}", extension).as_str()) {
```
[the remark this answers is not in the document]
Comment (sebastian-wehner-bruker) [reply src/file_readers/spectrum_readers/mini_tdf_reader.rs:31]: no, the metadata file is called:  *.globalmetadata.parquet

File: src/file_readers/spectrum_readers/mini_tdf_reader.rs [hunk 2]
```diff
@@ -23,6 +24,18 @@ pub struct MiniTDFReader {
...
+            return filename
+        }
+    }
+    panic!("No '*.ms2spectrum.{}' file found in '{}'", extension, ms2_dir_path)
```
[the remark this answers is not in the document]
Comment (sebastian-wehner-bruker) [reply src/file_readers/spectrum_readers/mini_tdf_reader.rs]: done. function is throwing an error now, which is unwrapped in the calling function.

**Comment by sebastian-wehner-bruker** [comment]
I made a change in the function checking for file formats: As miniTDF does not have a defined extension for the parent directory (other than tdf, which has the ".d" extension). The assumption should be that the provided directory should either be a tdf or minitdf. No other file formats are supported for reading anyways. 
This is also super helpful for the ProteoScape workflow usecase, where no requirements for the directory name for miniTDF files are being made.

**Pull request merged**